emnify/modules/device/api_call_manager.py

```diff
@@ -65,8 +66,12 @@ class UpdateDevice(BaseApiManager):
     response_handlers = {
         204: 'return_success',
         401: 'unauthorised',
+        422: 'process_exception'
```

Comment:
What do you think of placing it as default processing for the code and overwrite it where it is needed?